package.json

```diff
@@ -1,34 +1,33 @@
 {
-	"name": "@coder/logger",
-	"description": "Beautiful logging inspired by https://github.com/uber-go/zap.",
-	"scripts": {
-		"prepare": "tsc",
-		"test": "jest"
-	},
-	"version": "1.1.16",
-	"main": "out/index.js",
-	"types": "out/index.d.ts",
-	"author": "Coder",
-	"license": "MIT",
-	"peerDependencies": {
-		"@google-cloud/logging": "^4.5.2"
-	},
-	"devDependencies": {
-		"@types/jest": "^24.0.16",
-		"@types/node": "^12.6.8",
-		"jest": "^24.8.0",
-		"ts-jest": "^24.0.2",
-		"tslint": "^5.18.0",
-		"typescript": "^3.5.3"
-	},
-	"jest": {
-		"moduleFileExtensions": [
-			"js",
-			"ts"
-		],
-		"transform": {
-			"^.+\\.tsx?$": "ts-jest"
-		},
-		"testRegex": ".*\\.test\\.ts"
-	}
+  "name": "@coder/logger",
+  "description": "Beautiful logging inspired by https://github.com/uber-go/zap.",
+  "scripts": {
+    "lint": "eslint src/**",
+    "prepare": "tsc",
+    "test": "mocha --require ts-node/register ./test/*.test.ts"
```

Comment:
non-blocking: more so curious why we're switching from Jest to Mocha? I don't care, just curious 😄

Reply:
It's not so much a switch here since there were no tests before but honestly I'm just familiar with Mocha so I didn't want to do research on how to set up Jest haha.